recipe/meta.yaml

```
@@ -1,39 +1,41 @@
 {% set name = "sentencepiece" %}
-{% set version = "0.1.91" %}
+{% set version = "0.1.95" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
-source: 
-  url: https://github.com/google/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: acbc7ea12713cd2a8d64892f8d2033c7fd2bb4faecab39452496120ace9a4b1b
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz  # [unix]
+  url: https://github.com/google/{{ name }}/archive/v{{ version }}.tar.gz  # [win]
+  sha256: 8dd6e3e110f4c3f85a46e4a2ae1b6f7cf020b907fab50eac22beccf1680e0ea5  # [unix]
+  sha256: 1c0bd83e03f71a10fc934b7ce996e327488b838587f03159fd392c77c7701389  # [win]
 
 build:
-  number: 4
-  skip: True  # [py<36 or osx]
+  number: 0
+  skip: True  # [py<36]
+  script: {{ PYTHON }} -m pip install . -vv  # [unix]
 
 requirements:
   build:
     - cmake
     - {{ compiler('cxx') }}
-    - pkg-config
     - gperftools  # [unix]
     - make
+    - pkg-config
   host:
     - pip
     - python
-    - gperftools  # [unix]
   run:
     - python
-    - gperftools  # [unix]
 
 test:
   imports:
     - sentencepiece
   commands:
```

Running the upstream test suite (if it is in the sources) can be as easy as:
```suggestion
  requires:
    - pytest
  source_files:
    - tests
  commands:
    - pytest tests
```